src/ui/component/userEmailNew/view.jsx

```diff
@@ -39,12 +46,25 @@ function UserEmailNew(props: Props) {
           className="form-field--short"
           placeholder={__('hotstuff_96@hotmail.com')}
           type="email"
-          id="sign_up_email"
+          name="sign_up_email"
           label={__('Email')}
           value={newEmail}
           error={errorMessage}
           onChange={e => setEmail(e.target.value)}
         />
+        <FormField
+          type="checkbox"
+          name="sync_checkbox"
+          label={__('Sync your balance between devices')}
```

Comment:
This will be required for settings, right? [at least in the near future]

If so, use: Sync balance and preferences across devices

Also, on lbry.tv I do not think we even need to present this option, it should just be automatically true.